applications/nrf5340_audio/tools/buildprog/buildprog.py

```diff
@@ -51,6 +51,16 @@
 COMP_FLAG_RELEASE = NRF5340_AUDIO_FOLDER / "overlay-release.conf"
```

Comment:
Have the changed code been run through a Python code formatter? I know that the rest of buildprog.py has.

Reply:
Like autopep8

Reply:
I have run buildprog.py/fw_info_data.py/pcft_sign.py with  pylint.
_pylint --rcfile=$ZEPHYR_BASE/scripts/ci/pylintrc_
Will run with autopep8 again.